src/common/GlobalStyles.js

```diff
@@ -29,18 +29,21 @@ export default createGlobalStyle`
     font-size: 112%;
```

Comment:
In a later step maybe you can commit to _one_ size-unit, e.g. `rem`. This lets you easily change values globally and provides a better performance on different devices.